pkg/api/v1alpha1/cloudstackdatacenterconfig_types.go

```diff
@@ -45,6 +45,19 @@ type CloudStackResourceIdentifier struct {
 	Name string `json:"name,omitempty"`
 }
 
+func (r *CloudStackResourceIdentifier) Equals(o *CloudStackResourceIdentifier) bool {
```

Comment:
nit: I think we have been using `Equal` as the standard

Reply:
renamed